### ISSUE TITLE
Exception occurred while trying to compute conventioned absolute path (java.lang.NullPointerException)

### ISSUE THREAD

**Issue by WhyJustMe** [burningwave/core#41]
The Example "[Finding all classes that have package name that matches a regex](https://www.burningwave.org/how-to-find-all-classes-in-a-package/)" leads to the following exception. The path "C:/xxx..." (_product name x-ed out_) does not exist in the file system and was probably created by Burningwave.

```
[WARN ]  FileSystemItem                       -  (FileSystemItem.java:619) - Exception occurred while trying to compute conventioned absolute path of C:/xxxxxxxxxxxxxx/bin/default. Trying to repeat the operation.
[ERROR]  TaskBasedIterator                    -  (TaskBasedIterator.java:295) - Exception occurred
java.lang.NullPointerException: null
	at org.burningwave.core.io.FileSystemItem.lambda$isChildOf$22(FileSystemItem.java:553)
	at org.burningwave.core.io.FileSystemItem.computeConventionedAbsolutePathAndExecute(FileSystemItem.java:616)
	at org.burningwave.core.io.FileSystemItem.computeConventionedAbsolutePathAndExecute(FileSystemItem.java:610)
	at org.burningwave.core.io.FileSystemItem.isChildOf(FileSystemItem.java:552)
	at org.burningwave.core.classes.PathScannerClassLoader.hasBeenCompletelyLoaded(PathScannerClassLoader.java:257)
	at org.burningwave.core.classes.ClassPathScanner$Abst.scanAndAddToPathScannerClassLoader(ClassPathScanner.java:210)
	at org.burningwave.core.classes.ClassPathScanner$Abst.lambda$findBy$0(ClassPathScanner.java:164)
	at org.burningwave.core.iterable.IterableObjectHelperImpl$Iterator.lambda$buildOutputCollectionHandler$1(IterableObjectHelperImpl.java:802)
	at org.burningwave.core.classes.ClassPathScanner$Abst.lambda$findBy$1(ClassPathScanner.java:161)
	at org.burningwave.core.iterable.TaskBasedIterator.lambda$iterate$2(TaskBasedIterator.java:142)
	at org.burningwave.core.iterable.TaskBasedIterator.consume(TaskBasedIterator.java:293)
	at org.burningwave.core.iterable.TaskBasedIterator.iterate(TaskBasedIterator.java:159)
	at org.burningwave.core.iterable.IterableObjectHelperImpl.iterateAndGet(IterableObjectHelperImpl.java:640)
	at org.burningwave.core.classes.ClassPathScanner$Abst.lambda$findBy$3(ClassPathScanner.java:153)
	at org.burningwave.core.classes.SearchContext.executeSearch(SearchContext.java:96)
	at org.burningwave.core.classes.ClassPathScanner$Abst.findBy(ClassPathScanner.java:148)
	at my.playground.burningwave.Classes.find(Classes.java:21)
	at my.playground.burningwave.Classes.main(Classes.java:30)
```

The `burningwave.static.properties` file as follows:
```
banner.hide=true
priority-of-this-configuration=100
```

Dependency for Burningwave in `build.gradle` (earlier versions show the same error):
```
dependencies {
	testImplementation group: 'org.burningwave', name: 'core', version: '12.65.1'
```

IDE
```
Eclipse IDE for Enterprise Java and Web Developers

Version: 2023-12 (4.30.0)
Build id: 20231201-2043
```

Sample program used: [Classes.java.txt](https://github.com/user-attachments/files/16746299/Classes.java.txt)

**Comment by Roberto-Gentili** [comment]
Could you try to call `org.burningwave.core.assembler.StaticComponentContainer.Cache.clear(true)` before  you execute the search and tell me if you solved?

**Comment by Roberto-Gentili** [comment]
Closed with the release of version 12.65.2
